payu_config/archive_scripts/build_intake_ds.py

```diff
@@ -22,7 +22,7 @@
 
 METADATA_FILENAME = "metadata.yaml"
 UUID_FIELD = "experiment_uuid"
-ARCHIVE_PATH = "archive"
+ARCHIVE_PATH = "archive/"
```

Comment:
```suggestion
ARCHIVE_PATH = "archive"
```

In the output file, the above suggestion produces `.../archive/intake_esm_ds.json`, instead of `.../archive//intake_esm_ds.json`